h5py/tests/test_base.py

```diff
@@ -80,3 +143,12 @@ def test_file(self):
                 os.unlink(fname)
             except Exception:
                 pass
+
+def test_is_hdf5():
+    filename = File(tempfile.mktemp(), "w").filename
+    fid = is_hdf5(filename)
+    assert fid is True
```

Comment:
```suggestion
    assert is_hdf5(filename)
```

The name `fid` is a bit confusing - normally I'd guess that means 'file ID'.